crates/backend/Cargo.toml

```
@@ -17,7 +17,7 @@ version = "0.2.101"
 extra-traits = ["syn/extra-traits"]
 
 [dependencies]
-bumpalo = "3.0.0"
+bumpalo = "3.19.0"
```

Ideally we don't want to specify compatible versions unless we are setting the minimal version required.

Equally, we don't want to raise the minimal version required unless actually necessary.

```suggestion
bumpalo = "3"
```